### ISSUE TITLE
Regression loss is 0

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [fizyr/keras-retinanet#1110]
Hello,

I am trying to figure out where my model does not work well by calculating the loss for each training patch. I found predicted bounding box does not have 100% iou to groundtruth bouning box, but the regression loss is still zero. 
Any help?

best.

![Screenshot from 2019-09-04 11-55-37](https://user-images.githubusercontent.com/26878250/64245366-1545da00-cf0b-11e9-8225-91822e76b897.png)

**Comment by hgaiser** [comment]
If the regression loss is zero, it most likely means that there were no "approved" annotations. Annotations have to be big enough for them to be used in training. Try visualizing your dataset using `debug.py` to see if your annotations are used during training.

**Comment by HuangBo-Terraloupe** [comment]
So you mean for this patch, there is no groundtruth bounding box in the annotation file. I will use debug.py and give feed back.
Thanks for the help.

**Comment by HuangBo-Terraloupe** [comment]
I find out the problem because of miss annotations. thanks, problem solved.
